notifications/broadcast.py

```diff
@@ -243,7 +239,7 @@ def has_expired(cert_instance, noti_rule_msg):
     relationship = str(noti_rule_msg["relationship"])
 
     subject_list.append("Alert - %s" % (relationship))
-    subject_list.append("\nAn SSL Cert on %s " % \
+    subject_list.append("\tAn SSL Cert on %s " % \
```

Comment:
An SSL certificate.... what is this cert business? cert is not a word in the english language